CHANGELOG.md

```diff
@@ -6,10 +6,16 @@ All notable changes to [diagram-js](https://github.com/bpmn-io/diagram-js) are d
 
 _**Note:** Yet to be released changes appear here._
 
-### Breaking Changes
+## 11.0.0
 * `FEAT`: new popupMenu UI featuring menu and group titles, search, entry descriptions and documentation urls ([#687](https://github.com/bpmn-io/diagram-js/pull/687))
 * `FEAT`: introduction of `.djs-parent` class to canvas and popup menu root ([#687](https://github.com/bpmn-io/diagram-js/pull/687))
+* `CHORE`: provide UI utilities through `diagram-js/lib/ui` ([#692](https://github.com/bpmn-io/diagram-js/pull/692))
+* `DOCS`: update popup menu provider example ([#678](https://github.com/bpmn-io/diagram-js/pull/678))
+* `DEPS`: bump utility dependencies
 
+### Breaking Changes
+* Improved popup menu UI and behaviour, resulting in the modification of some of the composing HTML elements and CSS classes.
+* The popup menu now exists outside of the canvas container. The `djs-parent` class was introduced to the canvas and popup menu roots.
```

Comment:
Let's answer here two questions:
1) What is breaking?
2) How do I migrate to new version?